flow-typed/npm/chai_v4.x.x.js

```diff
@@ -0,0 +1,459 @@
+    static isNotBoolean(val: mixed, msg?: string): void;
+
+    static typeOf(val: mixed, type: string, msg?: string): void;
+    static notTypeOf(val: mixed, type: string, msg?: string): void;
+
+    static instanceOf(val: mixed, constructor: Class< * >, msg?: string): void;
+    static notInstanceOf(val: mixed, constructor: Class< * >, msg?: string): void;
+
+    static include(exp: string, inc: mixed, msg?: string): void;
+    static include<T>(exp: Array<T>, inc: T, msg?: string): void;
+
+    static notInclude(exp: string, inc: mixed, msg?: string): void;
+    static notInclude<T>(exp: Array<T>, inc: T, msg?: string): void;
+
+    static match(exp: mixed, re: RegExp, msg?: string): void;
+    static notMatch(exp: mixed, re: RegExp, msg?: string): void;
+
+    static property(obj: Object, prop: string, msg?: string): void;
+    static notProperty(obj: Object, prop: string, msg?: string): void;
+    static deepProperty(obj: Object, prop: string, msg?: string): void;
+    static notDeepProperty(obj: Object, prop: string, msg?: string): void;
+
+    static propertyVal(
+      obj: Object,
+      prop: string,
+      val: mixed,
+      msg?: string
+    ): void;
+    static propertyNotVal(
+      obj: Object,
+      prop: string,
+      val: mixed,
+      msg?: string
+    ): void;
+
+    static deepPropertyVal(
+      obj: Object,
+      prop: string,
+      val: mixed,
+      msg?: string
+    ): void;
+    static deepPropertyNotVal(
+      obj: Object,
+      prop: string,
+      val: mixed,
+      msg?: string
+    ): void;
+
+    static lengthOf(exp: mixed, len: number, msg?: string): void;
+
+    static throws<E>(
+      func: () => any,
+      err?: Class<E> | Error | RegExp | string,
+      errorMsgMatcher?: string | RegExp,
+      msg?: string
+    ): void;
+    static doesNotThrow<E>(
+      func: () => any,
+      err?: Class<E> | Error | RegExp | string,
+      errorMsgMatcher?: string | RegExp,
+      msg?: string
+    ): void;
+
+    static closeTo(
+      actual: number,
+      expected: number,
+      delta: number,
+      msg?: string
+    ): void;
+    static approximately(
+      actual: number,
+      expected: number,
+      delta: number,
+      msg?: string
+    ): void;
+
+    // chai-immutable
+    static sizeOf(val: mixed, length: number): void;
+
+    // chai-as-promised
+    static eventually: PromisedAssert;
+    static isFulfilled(promise: Promise<any>, message?: string): Promise<void>;
+    static becomes(promise: Promise<any>, expected: any, message?: string): Promise<void>;
+    static doesNotBecome(promise: Promise<T>, expected: any, message?: string): Promise<void>;
+    static isRejected<T>(promise: Promise<T>, message?: string): Promise<void>;
+    static isRejected<T>(promise: Promise<T>, expected: any, message?: string): Promise<void>;
+    static isRejected<T>(promise: Promise<T>, match: RegExp, message?: string): Promise<void>;
+    static notify(fn: Function): Promise<void>;
+  }
```

Comment:
Lines 291-299 were copied from the typescript type definitions and modified accordingly.